src/OnlineRetailPortal.Contracts/Contracts/IImageWriterResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Contracts.Contracts
+{
+    public interface IImageWriterResponse
```

Comment:
why is this an interface ?? looks like a plain class to me. Do you have any specific use case

Reply:
Yes, I interfaced the writer as we can add another writer that will directly write to the Database, if needed in the future, without changing any code here.
I also interfaced the image handler in case we need to add a completely different way of handling (writing and deleting)  files.